mino/minogrpc/server.go

```diff
@@ -666,12 +672,41 @@ func (mgr *connManager) Acquire(to mino.Address) (grpc.ClientConnInterface, erro
 	}
 
 	// Connecting using TLS and the distant server certificate as the root.
-	conn, err = grpc.Dial(netAddr.GetDialAddress(),
+	addr := netAddr.GetDialAddress()
+	tracer, err := tracing.GetTracerForAddr(addr)
+	if err != nil {
+		return nil, err
+	}
+
+	decorator := func(ctx context.Context, span opentracing.Span, method string,
+		req, resp interface{}, grpcError error) {
+		md, ok := metadata.FromOutgoingContext(ctx)
+		if !ok {
+			return
+		}
+		protocol := getOrEmpty(md, tracing.ProtocolTag)
+		if protocol != "" {
+			span.SetTag(tracing.ProtocolTag, protocol)
+		}
+
+		streamID := getOrEmpty(md, HeaderStreamIDKey)
+		if streamID != "" {
+			span.SetTag(HeaderStreamIDKey, streamID)
+		}
+	}
```

Comment:
does this function need to be declared inline? I'm not sure I see why.
Could we deport it outside this function, again for readability?